Polyhedron/demo/Polyhedron/Plugins/Surface_mesh_deformation/Scene_edit_polyhedron_item.cpp

```diff
@@ -1220,6 +1309,10 @@ void Scene_edit_polyhedron_item::reset_spheres()
 {
   d->spheres = NULL;
 }
+void Scene_edit_polyhedron_item::reset_spheres_ctrl()
+{
+  d->spheres_ctrl = NULL;
+}
```

Comment:
(A general note for everybody modifying the Polyhedron demo. cc @sloriot)
The new API of [`QObject::connect`][c] allows to connect a signal to a functor. There is no longer the needs to add slots, and this modify the the `.h`.

If C++11 was allowed, the connection would be really simple, with a lambda expression:
```C++
connect(d->spheres_ctrl, &QObject::destroyed, this, [this] () { this->d->spheres_ctrl = NULL} );
```
The expression `[this] () { this->d->spheres_ctrl = NULL}` is a lambda-expression, that captures the value of `this`.

Without C++11, it can be done almost as easily:
```C++
struct Reset_spheres_ctrl {
  Scene_edit_polyhedron_item_priv* d;
  Reset_spheres_ctrl(Scene_edit_polyhedron_item_priv* d) : d(d) {}
  void operator()() const { d->spheres_ctrl = NULL; }
};
// [...]
connect(d->spheres_ctrl, &QObject::destroyed, this, Reset_spheres_ctrl(d) );
```
[c]: http://doc.qt.io/qt-5/qobject.html#connect-5

See also [New Signal Slot Syntax in Qt 5, in the wiki of Qt][*]. @maxGimeno: please read that document. You will see that it is really interesting.
[*]: https://wiki.qt.io/New_Signal_Slot_Syntax

I really would like that we allow to use C++11 in our demos. With CMake>=3.1, that is as simple as:
```CMake
target_compile_features(my_plugin PRIVATE cxx_lambdas)
```